data/services/aws-rds.toml

```diff
@@ -1,3 +1,5 @@
+# Service metadata lives
+# https://github.com/18F/aws-broker/blob/59f949b/catalog-template.yml#L8
```

Comment:
Interesting point! I think in general if the file moves, we should update the links in these comments - my hope would be that the editor of this file would be able to hop over to the referenced file and edit the description to match.